sdklab/regressions/regression_github_990.py

```diff
@@ -0,0 +1,284 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import asyncio
+import logging
+import functools
+import ssl
+from test_utils import test_env, logging_hook
+from azure.iot.device.aio import IoTHubDeviceClient
+import azure.iot.device.common
+
+
+logging.basicConfig(level=logging.WARNING)
+logging.getLogger("azure.iot").setLevel(level=logging.DEBUG)
+
+"""
+This app simulates was a bug that was reported where a websockets connection was raising a
```

Comment:
re: filenaming here - I'm not sure "regression_github_990" is a particularly great name, given that it requires knowing what GH990 was in order to know what to expect. I appreciate this long block comment here explaining the issue, that's fantastic, but you still have to open the file to see it, and as we add more these could get hard to identfy

On the other hand, I realize that it's a highly specific problem that might be hard to encapsulate in a filename.

Not requesting a change here because I don't actually have a better idea. Perhaps this is ideal after all, as having long verbose titles doesn't do much for a quick lookup either.